docs/custom_fields.rst

```diff
@@ -95,40 +95,6 @@ Both :class:`Function <marshmallow.fields.Function>` and :class:`Method <marshma
     result = schema.load({"balance": "100.00"})
     result["balance"]  # => 100.0
 
-.. _adding-context:
-
-Adding Context to `Method` and `Function` Fields
```

Comment:
thought: should we replace this with the equivalent using the new `Context` API?

Reply:
#2727 addresses this